### ISSUE TITLE
Comments and Discussions section

### ISSUE THREAD

**Issue by amrit03b** [amrit03b/TravelTales#16]
Enable users to leave comments on public travel journals, fostering discussions and interactions among the community.

**Comment by seabeePraveen** [comment]
hello @amrit03b i am very interested in working with issue please assign me to this task

**Comment by amrit03b** [comment]
> hello @amrit03b i am very interested in working with issue please assign me to this task

Assigned @seabeePraveen

**Comment by seabeePraveen** [comment]
@amrit03b i think for this section it might need backend to store the data what do u want me to do?

**Comment by amrit03b** [comment]
> @amrit03b i think for this section it might need backend to store the data what do u want me to do?

Yes, do create both frontend and backend for the comment section.
